ngraph/core/reference/include/ngraph/runtime/reference/convolution_backprop_data.hpp

```diff
@@ -242,55 +325,106 @@ namespace ngraph
+                    conv_input_data = &extended_input[0];
                 }
-                // Compute backward pad out pad above
-                CoordinateDiff backward_delta_out_pad_above;
-                backward_delta_out_pad_above.resize(spatial_dim_count);
 
-                for (size_t i = 0; i < spatial_dim_count; i++)
+                const size_t dilation_dim = std::accumulate(
+                    filter_dilation.begin(), filter_dilation.end(), 1, std::multiplies<size_t>());
+                if (dilation_dim >= 2)
                 {
-                    backward_delta_out_pad_above[i] =
-                        (static_cast<ptrdiff_t>(filter_shape[i + 2]) - 1) * filter_dilation[i] +
-                        ((forward_in_pad_bellow[i] + ((in_shape[i + 2]) - 1) * in_dilation[i] +
-                          forward_in_pad_above[i] -
-                          (static_cast<ptrdiff_t>(filter_shape[i + 2]) - 1) * filter_dilation[i]) %
-                         stride[i]) -
-                        forward_in_pad_above[i];
+                    extend_with_zeros<T>(filter_dilation,
+                                         filter_shape,
+                                         reinterpret_cast<const T*>(&reversed[0]),
+                                         conv_filter_shape,
+                                         extended_filter);
+                    std::fill(conv_filter_dilation.begin(), conv_filter_dilation.end(), 1);
+                    conv_filter_data = &extended_filter[0];
                 }
 
-                convolution_backprop_impl<OUTPUT, FILTER, INPUT, ACCUMULATION>(
-                    delta_out,
-                    &reversed[0],
-                    delta_in,
-                    out_shape,
-                    filter_shape,
-                    in_shape,
-                    in_dilation,
-                    filter_dilation,
-                    backward_delta_out_pad_below,
-                    backward_delta_out_pad_above,
-                    stride,
-                    0,
-                    1,
-                    filter_out_channel_axis,
-                    filter_in_channel_axis,
-                    0,
-                    1);
+                convolution_backprop_impl(conv_input_data,
+                                          conv_filter_data,
+                                          delta_out,
+                                          conv_input_shape,
+                                          conv_filter_shape,
+                                          out_shape,
+                                          conv_stride,
+                                          conv_filter_dilation,
+                                          forward_in_pad_bellow,
+                                          forward_in_pad_above,
+                                          output_padding);
+            }
+
+            // DEPRECATED, can't be removed currently due to arm-plugin dependency
```

Comment:
You can use `NGRAPH_DEPRECATED` macro to deprecate any methods

Reply:
Fixed.